### PR TITLE
fixed errors and inconsistencies in uk locale

### PR DESCRIPTION
**Pull request by JustACuteGirlLookingForAManToFixHerBugs**

### CODE REVIEW

**Comment by ghostwords** [comment]
Thank you! I'll take a look next week.

**Comment by a-polivanchuk** [comment]
I've reviewed the proposed translations updates and couldn't approve them.
Most of the suggestions are uncommon and even mistaken for Ukrainian language.
I would recommend to close this PR and propose any suggestions on the official Transifex.
Thanks!

File: src/_locales/uk/messages.json [hunk 1]
```diff
@@ -10,15 +10,15 @@
         }
     },
     "invalid_json": {
-        "message": "Неправильний файл JSON.",
+        "message": "Невірний файл JSON.",
```
Comment (a-polivanchuk) [inline src/_locales/uk/messages.json:13]: "Невірний" is not correct translation.

File: src/_locales/uk/messages.json [hunk 2]
```diff
@@ -28,35 +28,35 @@
...
+        "message": "Перевіряти, чи <a target='_blank' href='https://www.eff.org/privacybadger/faq#What-is-a-third-party-tracker'>сторонні домени</a> виконують <a target='_blank' href='https://www.eff.org/dnt-policy'>політику EFF щодо відмови від стеження</a>",
         "description": "Checkbox label on the general settings page"
     },
     "donate_to_eff": {
         "message": "Зробити внесок для EFF",
         "description": "Button shown in the popup and on the intro page."
     },
     "intro_donate_subheading": {
-        "message": "Допоможіть нам, зробивши внесок і виявивши свою підтримку наших засобів",
+        "message": "Допоможіть нам, зробивши внесок і виявивши свою підтримку наших інструментів",
```
Comment (a-polivanchuk) [inline src/_locales/uk/messages.json:43]: synonyms

File: src/_locales/uk/messages.json [hunk 3]
```diff
@@ -88,11 +88,11 @@
         "description": "Dropdown control setting on the Tracking Domains options page tab."
     },
     "options_webrtc_setting": {
-        "message": "Не дозволяти WebRTC виявляти локальну IP-адресу",
+        "message": "Не дозволяти WebRTC виявляти вашу локальну IP-адресу",
```
Comment (a-polivanchuk) [inline src/_locales/uk/messages.json:91]: added back unnecessary omitted "your"

File: src/_locales/uk/messages.json [hunk 4]
```diff
@@ -104,7 +104,7 @@
         "description": "This is an options page tab heading."
     },
     "share_button_title_twitter": {
-        "message": "Поширити у Twitter",
+        "message": "Поширити у Твітері",
```
Comment (a-polivanchuk) [inline src/_locales/uk/messages.json:107]: It's better keep the brand names like Facebook, Firefox, Mozilla, Twitter untranslated

File: src/_locales/uk/messages.json [hunk 5]
```diff
@@ -186,15 +186,15 @@
...
+        "message": "Увімкнення навчання у приватних/анонімних вікнах може залишати сліди історії перегляду на комп'ютері. Зазвичай, Privacy Badger буде блокувати відомі йому елементи стеження в приватних/анонімних вікнах, але не буде вивчати нові. Ймовірно, ви захочете увімкнути цю функцію, якщо багато часу проводите у приватному/анонімному перегляді.",
         "description": "Tooltip on the general settings page"
     },
     "learning_prompt_text1": {
         "message": "Privacy Badger відтепер типово не навчатиметься під час перегляду.",
         "description": "First part of a prompt to visit the blog."
     },
     "learning_prompt_text2": {
-        "message": "Натисніть внизу, щоб дізнатися більше.",
+        "message": "Натицніть внизу, щоб дізнатися більше.",
```
Comment (a-polivanchuk) [inline src/_locales/uk/messages.json:197]: "Натицніть" invalid word in Ukrainian language

File: src/_locales/uk/messages.json [hunk 6]
```diff
@@ -210,7 +210,7 @@
         "description": "Tooltip that comes up when you hover over the 'tracking domains' link under the Tracking Domains tab on the options page."
     },
     "intro_report_button": {
-        "message": "Будь ласка, не забудьте натиснути на \"Privacy Badger пошкодив цей сайт? Повідомте нас!\". Ми поважаємо вашу приватність, і тому не надсилаємо автоматичні звіти.",
+        "message": "Будь ласка, не забудьте натицьнути на \"Privacy Badger пошкодив цей сайт? Повідомте нас!\". Ми поважаємо вашу приватність, і тому не надсилаємо автоматичні звіти.",
```
Comment (a-polivanchuk) [inline src/_locales/uk/messages.json:213]: "натицьнути" invalid word in Ukrainian language

File: src/_locales/uk/messages.json [hunk 7]
```diff
@@ -270,11 +270,11 @@
         "description": "First part of a reminder to visit the intro page. Shown in popup until the user clicks on the reminder link or browses through the intro page."
     },
     "intro_text2": {
-        "message": "Щоб дізнатися, як працює Privacy Badger, натисніть внизу для швидкого ознайомлення.",
+        "message": "Щоб дізнатися, як працює Privacy Badger, натицьніть внизу для швидкого ознайомлення.",
```
Comment (a-polivanchuk) [inline src/_locales/uk/messages.json:273]: "натицьніть" invalid word in Ukrainian language

File: src/_locales/uk/messages.json [hunk 8]
```diff
@@ -545,15 +545,15 @@
         "description": ""
     },
     "share_button_title_facebook": {
-        "message": "Поширити на Facebook",
+        "message": "Поширити на Фейсбуці",
```
Comment (a-polivanchuk) [inline src/_locales/uk/messages.json:548]: It's better keep the brand names like Facebook, Firefox, Mozilla, Twitter untranslated

**Comment by a-polivanchuk** [comment]
I suppose that corrections have been made based on the context rather then on the source English strings.

**Comment by ghostwords** [comment]
Thank you @a-polivanchuk! Closing as per https://github.com/EFForg/privacybadger/pull/2763#issuecomment-826555593.

**Pull request closed without merging**